worker.py

```diff
@@ -251,7 +256,7 @@ def get_status_message(workers, count, start_time, points_stats):
         '',
     ]
     previous = 0
-    for i in range(4, count + 4, 4):
+    for i in range(5, count + 5, 5):
```

Comment:
Maybe this should be a variable, in `config.py` which could be set, and have a reasonable default (which limits its width to 80 chars?).